server.js

```diff
@@ -14,12 +14,68 @@
  * limitations under the License.
  */
 
-const express = require("express");
+const express = require('express');
+const locale = require('locale');
+const path = require('path');
+const fs = require('fs');
+const cookieParser = require('cookie-parser');
+
 const app = express();
 
-app.use(express.static("dist"));
-// app.use(express.static("dist/en"));
-app.use(express.static("dist/ja"));
+const staticFilesPath = [
+  'images', 'test', 'app.css', 'app.js'
+];
+
+// Detect all locale directories
+const supportedLocales = [];
+const dirs = fs.readdirSync(path.join(__dirname, 'dist'));
```

Comment:
Do we actually need this? Could we allow the user to set any language (... checking that it's valid, as per my security comment below) and then try the file / fallback to 'en'?